### PR TITLE
Ditch support to mb wallet

### PR DESCRIPTION
**Pull request by rubenmarcus**
### **User description**
Bitte Wallet will be the standard now.

___

### **PR Type**
Enhancement, Tests

___

### **Description**
- Added `onlyBitteWallet` property to `BitteWalletContextProvider` and replaced all references from `onlyMbWallet` to `onlyBitteWallet`.
- Removed the entire Mintbase wallet context implementation and its associated tests.
- Removed exports and setup functions related to Mintbase wallet.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>BitteWalletContext.tsx</strong><dd><code>Update BitteWallet context to replace Mintbase wallet references</code></dd></summary>
<hr>

packages/react/src/BitteWalletContext.tsx

<li>Added <code>onlyBitteWallet</code> property to context provider.<br> <li> Replaced references from <code>onlyMbWallet</code> to <code>onlyBitteWallet</code>.<br> <li> Updated wallet setup function to use <code>setupBitteWallet</code>.<br>

</details>

  </td>
  <td><a href="https://github.com/Mintbase/mintbase-js/pull/541/files#diff-32ff94b49bbf55c9b1e775cccc56924eed4865eda1de13da559d5b705df13499">+8/-7</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>MintbaseWalletContext.tsx</strong><dd><code>Remove Mintbase wallet context implementation</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

packages/react/src/MintbaseWalletContext.tsx

- Removed Mintbase wallet context implementation.

</details>

  </td>
  <td><a href="https://github.com/Mintbase/mintbase-js/pull/541/files#diff-3739b90b841626f8f142fdbda174bc7452eaad94045278243370640f1ad27c9a">+0/-201</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>index.ts</strong><dd><code>Remove MintbaseWalletContext export</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

packages/react/src/index.ts

- Removed export for MintbaseWalletContext.

</details>

  </td>
  <td><a href="https://github.com/Mintbase/mintbase-js/pull/541/files#diff-1cee8646d2cfba37d6ce6a6e9a8d16f8caba0b99fc3a1ad0cb997ed8c7384d2e">+0/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>bitte-wallet.ts</strong><dd><code>Update BitteWallet setup to replace Mintbase wallet references</code></dd></summary>
<hr>

packages/react/src/wallet/bitte-wallet.ts

- Replaced references from `onlyMbWallet` to `onlyBitteWallet`.

</details>

  </td>
  <td><a href="https://github.com/Mintbase/mintbase-js/pull/541/files#diff-af5da20343ab9483a3cd055e4ef548b5250037c345772aee04ec419e2cb187b2">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>index.ts</strong><dd><code>Remove Mintbase wallet setup export</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

packages/wallet/src/index.ts

- Removed export for `setupMintbaseWallet`.

</details>

  </td>
  <td><a href="https://github.com/Mintbase/mintbase-js/pull/541/files#diff-deba9f6c2e4268f9b472aad8f136c431b997a6cb234410b11362c7698fd8f895">+0/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>mintbase-wallet.ts</strong><dd><code>Remove Mintbase wallet implementation</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

packages/wallet/src/mintbase-wallet.ts

- Removed Mintbase wallet implementation.

</details>

  </td>
  <td><a href="https://github.com/Mintbase/mintbase-js/pull/541/files#diff-9185fb91b52e06e702f448ca0818e8c83cce2dd7026334ab60f209a0d00f2e18">+0/-272</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>setup.ts</strong><dd><code>Remove Mintbase wallet setup function</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

packages/wallet/src/setup.ts

- Removed Mintbase wallet setup function.

</details>

  </td>
  <td><a href="https://github.com/Mintbase/mintbase-js/pull/541/files#diff-0c935be1f04a367d5f44e601b808ffecf34ec5f27061a5015be40f9cd9854593">+0/-54</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Tests</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>MintbaseWalletContext.test.tsx</strong><dd><code>Remove Mintbase wallet context tests</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

packages/react/src/MintbaseWalletContext.test.tsx

- Removed Mintbase wallet context tests.

</details>

  </td>
  <td><a href="https://github.com/Mintbase/mintbase-js/pull/541/files#diff-01c2de8930cf8d282fe9a52c1ebb88dcb0ad5e510832d4f1783f656203eb0838">+0/-195</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: packages/data/src/api/nearPrice/nearPrice.test.ts [hunk 1]
```diff
@@ -1,52 +0,0 @@
-import { nearPrice } from './nearPrice';
```
Comment (bh2smith) [inline packages/data/src/api/nearPrice/nearPrice.test.ts:1]: deleting a test seems unrelated and unwise.

**Review by microchipgnu — APPROVED** [review]
LGTM! 🥳

File: packages/react/README.md [hunk 2]
```diff
@@ -72,15 +72,15 @@ the default way of interacting with Mintbase Wallet is using the MintbaseWalletC
 
 ```typescript
 import "@near-wallet-selector/modal-ui/styles.css";
-import { MintbaseWalletContextProvider } from  '@mintbase-js/react'
```
Comment (bh2smith) [inline packages/react/README.md:75]: Seems like this should just be called `WalletContextProvider` - independent of the name. If it were, then we wouldn't have had to change anything here.

File: packages/react/package.json [hunk 3]
```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "build": "tsc",
     "watch": "tsc && tsc --watch & jest --watch --coverage",
-    "lint": "eslint . --fix --ext ts --ext tsx",
-    "test": "jest --coverage"
```
Comment (bh2smith) [inline packages/react/package.json:10]: why tho?

**Review by sainthiago — APPROVED** [review]

File: packages/react/src/BitteWalletContext.tsx [hunk 4]
```diff
@@ -73,21 +74,21 @@ export const BitteWalletContextProvider: React.FC<ContextProviderType> = ({
...
       callbackUrl,
-      isOnlyMbWallet,
+      isOnlyBitteWallet,
       selectedNetwork,
       selectedContract,
-      isOnlyMbWallet ? { additionalWallets } : undefined,
+      isOnlyBitteWallet ? { additionalWallets } : undefined,
       successUrl, failureUrl,
     );
   };
```
Comment (bh2smith) [inline packages/react/src/BitteWalletContext.tsx:88]: I think we could live without having the wallet name everywhere. Like why not just `setupWallet`? People know its the bitte wallet from the import.

File: packages/react/src/wallet/bitte-wallet.ts [hunk 5]
```diff
@@ -57,15 +57,15 @@ export const BitteWalletAuth = {
...
+    onlyBitteWallet = false,
     network?,
     contractAddress?,
     options?: { additionalWallets?: Array<WalletModuleFactory> },
     successUrl?: string,
     failureUrl?: string,
   ): Promise<WalletSelectorComponents> => {
 
-    if (onlyMbWallet === false) {
+    if (onlyBitteWallet === false) {
```
Comment (bh2smith) [inline packages/react/src/wallet/bitte-wallet.ts:68]: Not related to this Pr, but this entire if block is unnecessary and could probably be replaced with much more compact similar code as in react/WalletContext

File: packages/react/src/MintbaseWalletContext.tsx [hunk 6]
```diff
@@ -1,201 +0,0 @@
-import React, {
```
Comment (bh2smith) [inline packages/react/src/MintbaseWalletContext.tsx:1]: no delete tests.

File: packages/react/src/MintbaseWalletContext.test.tsx [hunk 7]
```diff
@@ -1,195 +0,0 @@
-/**
```
Comment (bh2smith) [inline packages/react/src/MintbaseWalletContext.test.tsx:1]: no delete tests.

**Review by bh2smith — CHANGES_REQUESTED** [review]
1. A bunch of tests are deleted that shouldn't be.
2. I propose removing the redundant "Bitte" from all the naming.

**Comment by bh2smith** [comment]
This shouldn't have been merged.

**Pull request merged**